### DISCUSSION TITLE
Auto-update/check update via net adapter?

### DISCUSSION THREAD

**Discussion by hakanaiseishin** [emukidid/swiss-gc#544] [Ideas]
Would it be possible to implement an auto/manual update check and/or download/install/reboot in Swiss with a network adapter installed, or is it already a function I am perhaps not aware of?

**Comment by emukidid** [comment]
There's an open issue to try it: https://github.com/emukidid/swiss-gc/issues/332

The problem is that there's no code in libOGC to resolve hosts via hostname/DNS lookup so it'd be a bit more involved than just writing something to hit a URL. There's also the issue of everything being HTTPS these days too.

Alternatives could be achieved if a HDMI solution or something were to add Wi-Fi to the console.
